.gitignore

```diff
@@ -1,2 +1,2 @@
 .idea
-config.json
\ No newline at end of file
+deploy.config.json
```

Comment:
This shouldn't be needed. The actual config file is local to the bot, not the deploy git

Reply:
We should probably drop the example file then and just keep the definition in the readme, since that implies the opposite

Reply:
I'm ok with that as long as we have a reasonable definition somewhere